src/components/Top-Bar.vue

```diff
@@ -21,26 +22,36 @@
 import { mapGetters, mapState, mapActions } from 'vuex'
 import pluginHelper from '../mixins/pluginHelper.js'
 import Avatar from './Avatar.vue'
+import Notifications from './Notifications.vue'
 
 export default {
   mixins: [
     pluginHelper
   ],
-  components: { Avatar },
+  components: {
+    Avatar,
+    Notifications
+  },
   methods: {
-    ...mapActions(['toggleSidebar']),
+    ...mapActions(['toggleSidebar', 'fetchNotifications']),
     parseApp (app) {
       const regex = /([^/]+[.js$])\w/g
       let matched = regex.exec(app)
       return matched[0]
     }
   },
   computed: {
-    ...mapGetters(['configuration', 'isSidebarVisible']),
+    ...mapGetters(['configuration', 'isSidebarVisible', 'activeNotifications']),
     ...mapState(['user']),
     extendNavbarRight () {
       return this.getPlugins('phoenixNavbarRight')
     }
+  },
+  created: function () {
+    this.fetchNotifications(this.$client)
+    setInterval(() => {
```

Comment:
this will call fetchNotifications exactly 2 times and not periodically.

see https://stackoverflow.com/a/8682723/1698403